speech-synthesis/README.md

```diff
@@ -1,27 +1,47 @@
 # Building Speech Synthesis for New Languages
 
 Speech synthesis is the task of generating speech from text. It is useful if you want to build a speech interface.
+For a speech synthesizer, voice recordings from one person is usually required for consistency and natural sounding output.
 
-## Collecting Data
+## 1. Collecting Data
 
 There are many sources for speech data that can be used to train systems. It'd be a good idea to take a look here to see if there is data that you can use.
 
 * [CMU Wilderness Corpus](https://github.com/festvox/datasets-CMU_Wilderness): This is a very broad corpus with 699 languages.
 
-## Creating Data
+If there is none, follow the steps in *[Creating Data](#2-creating-data)* section to create a corpus.
+## 2. Creating Data
+The goal here is to create a speech corpus of aligned speech recordings and their transcriptions to be used to build the synthesizer. 
+1. Gather textual sentences in your language. You should aim at getting atleast 1 million words. Keep in mind:
+   - The copyright of the data you have gathered; make sure it is sufficient for your usage and distribution needs.
```

Comment:
Both of these are good points, but may be confusing for beginners. Perhaps you could link some tutorials:
> https://aclanthology.org/L18-1202/
> https://stackoverflow.com/questions/64860/best-way-to-convert-text-files-between-character-sets